.changeset/thirty-poets-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"@twilio-paste/icons": minor
```

Comment:
include core?

Reply:
Icons is a peer to core, so it doesn't need to be included. Everything else is a dependency or _inside_ of core.